events/tgapool.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from datetime import datetime
+
+import aiohttp
+import config
+import discord
+import pymongo
+from discord.ext import commands, tasks
+
+
+mclient = pymongo.MongoClient(config.mongoHost, username=config.mongoUser, password=config.mongoPass)
+
+
+class TGAPool(commands.Cog):
+    def __init__(self, bot):
+        self.GUILD = 238080556708003851
+        self.EVENT_CHANNEL = 672550040979636244
+        self.ENDPOINT = 'https://switchcord.net/rewards'
+        self.BACKGROUND = 'the-game-awards'
+        self.TROPHIES_PREFIX = 'tga-'
+        self.TROPHIES = ['tga-gold', 'tga-silver', 'tga-bronze']
+
+        self.bot = bot
+        self.db = mclient.bowser.users
+        self.guild = self.bot.get_guild(self.GUILD)
+        self.event_channel = self.guild.get_channel(self.EVENT_CHANNEL)
+
+        self.response_check.start()
+
+    @commands.command(name='tgatrophygrant')
```

Comment:
Trophies are all manually pulled? User backgrounds are already getting user info from the user collection, so this could be combined with response_check without any additional db queries.

Reply:
Manually pulled, yes. I did try and combine the code, but with the extra trophy logic, the code seemed more confusing when both cases were handled at the same time in the same method.

Reply:
If that is the intended case, and immediacy to users isn't wanted, then we can do that. I'll let you merge as a confirmation